### ISSUE TITLE
regression in 2.1.2: ValidationError: PostCSS Loader Invalid Options

### ISSUE THREAD

**Issue by garthk** [postcss/postcss-loader#348]
### `Details`

In 2.1.2 but not 2.1.1 or previous, `postcss-loader` is throwing many errors of the form:
```
ERROR in ./node_modules/css-loader??ref--2-2!./node_modules/postcss-loader/lib??ref--2-3!./node_modules/sass-loader/lib/loader.js??ref--2-4!./src/components/example.scss
    Module build failed: ValidationError: PostCSS Loader Invalid Options    
    options['path'] is an invalid additional property
```

### `Error (Logs|Stacks)`

From a later long blob of errors, we can extract a stack:

```
ERROR in ./src/components/status-bar/status-bar.scss
Module build failed: ModuleBuildError: Module build failed: ValidationError: PostCSS Loader Invalid Options
options[\'path\'] is an invalid additional property
    at runLoaders (src/node_modules/webpack/lib/NormalModule.js:195:19)
    at src/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at src/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at runSyncOrAsync (src/node_modules/loader-runner/lib/LoaderRunner.js:143:3)
    at iterateNormalLoaders (src/node_modules/loader-runner/lib/LoaderRunner.js:229:2)
    at iterateNormalLoaders (src/node_modules/loader-runner/lib/LoaderRunner.js:218:10)
    at src/node_modules/loader-runner/lib/LoaderRunner.js:233:3
    at context.callback (src/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.asyncSassJobQueue.push [as callback] (src/node_modules/sass-loader/lib/loader.js:79:9)
    at Object.done [as callback] (src/node_modules/neo-async/async.js:7921:18)
    at options.success (src/node_modules/node-sass/lib/index.js:308:32)
 @ ./src/components/status-bar/status-bar.scss
 @ ./src/components/status-bar/index.tsx
 @ ./src/components/common/toolbar.tsx
 @ ./src/app.tsx
 @ ./src/index.tsx
```

### `Reproduction [Code]`

None yet extracted.

### `Environment`

Observed on Linux and macOS, and Node 8.x and 9.x, with `webpack` 3.11.0  and `postcss-loader` 2.1.2.

**Comment by Helena-Wong** [comment]
encounter similar problem
`Module build failed: ModuleBuildError: Module build failed: ValidationError: PostCSS Loader Invalid Options

options['minimize'] is an invalid additional property`

**Comment by AntonRatnick** [comment]
Same error with options['outputStyle']
linux
node - 8
webpack@3.11.0
postcss-loader@2.1.2

**Comment by evilebottnawi** [comment]
Sorry, just wait in near future will be revert and release

**Comment by ai** [comment]
I reverted `additionalProperties` and released 2.1.3
